splice/default_settings.py

```diff
@@ -41,11 +41,16 @@ class DefaultConfig(object):
 
     S3 = {
         "bucket": "moz-tiles-dev",
+        "content": "moz-content-dev",
```

Comment:
might want a separate bucket for original content. Added a note in model.py review